### PR TITLE
Add "why ns not in dropdown" section to FAQ

### PR DESCRIPTION
**Pull request by nrfox**
Fixes kiali/kiali#3145

### CODE REVIEW

**Comment by nrfox** [comment]
@lucasponce if you wouldn't mind taking a look at the "cache" section to make sure that's accurate

File: content/documentation/v1.33/faq/general/why-ns-not-in-dropdown.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Why isn't my namespace in the Namespace Selection dropdown?
+weight: 105
+---
+:icons: font
+
+When deploying Kiali with the Kiali operator, by default some namespaces are https://github.com/kiali/kiali-operator/blob/v1.33/roles/default/kiali-deploy/defaults/main.yml#L24-L28[excluded] from the list of namespaces provided by the API and UI. Kiali filters out these namespaces and you will not see them in the Namespace Selection dropdown. You can adjust which namespaces are excluded by setting the `spec.api.namespaces.exclude` field on the Kaili CR.
```
Comment (jshaughn) [inline content/documentation/v1.33/faq/general/why-ns-not-in-dropdown.adoc:7]: Kaili CR -> Kiali CR

**Review by jshaughn — CHANGES_REQUESTED** [review]
@nrfox Adding this to the `/v1.33` content will make it immediately available for users but it will only be in that version going forward.  You also want to duplicate it to `/staging` to ensure it sows up in future versions.

Content looks good, deferring to Lucas for the cache.

**Comment by lucasponce** [comment]
I'd also link to the description of the CR for details
https://github.com/kiali/kiali-operator/blob/v1.33/deploy/kiali/kiali_cr.yaml#L746

The cache of objects is mostly transparent for the user, but the cache of namespaces it may be visible, in the meaning that if you create or delete a namespace, cache needs to expire to get a fresh data.

That expiration time is shorter than the regular parameters.

In short, I'd add a note with that link for further details, but the description/wording is ok.

**Comment by nrfox** [comment]
> I'd also link to the description of the CR for details
> https://github.com/kiali/kiali-operator/blob/v1.33/deploy/kiali/kiali_cr.yaml#L746
> 
> The cache of objects is mostly transparent for the user, but the cache of namespaces it may be visible, in the meaning that if you create or delete a namespace, cache needs to expire to get a fresh data.
> 
> That expiration time is shorter than the regular parameters.
> 
> In short, I'd add a note with that link for further details, but the description/wording is ok.

I did not realize there was a separate cache for namespaces. I thought that namespaces shared the same cache as the other kube objects. I'll get that updated. Thanks for taking a look!

**Review by jshaughn — APPROVED** [review]
LGTM

**Review by lucasponce — APPROVED** [review]
